### PR TITLE
Added runtime tests for TTY

### PR DESCRIPTION
**Pull request by DrDeano**
Removed types
Styling

### CODE REVIEW

File: test/mock/kernel/vga_mock.zig [hunk 1]
```diff
@@ -69,11 +69,11 @@ pub fn init() void {
 
 // User defined mocked functions
 
-pub fn mock_entryColour(fg: u4, bg: u4) u8 {
+pub fn orig_entryColour(fg: u4, bg: u4) u8 {
```
Comment (SamTebbs33) [inline test/mock/kernel/vga_mock.zig]: What's up with the renaming?

File: src/kernel/tty.zig [hunk 2]
```diff
@@ -361,55 +387,58 @@ fn writeString(str: []const u8) PrintError!void {
...
-///     IN format: []const u8 - The format string to print
-///     IN args: ...          - The arguments to be used in the formatted string
+///     IN comptime format: []const u8 - The format string to print
+///     IN args: ...                   - The arguments to be used in the formatted string
 ///
 pub fn print(comptime format: []const u8, args: ...) void {
     // Printing can't error because of the scrolling, if it does, we have a big problem
-    fmt.format({}, PrintError, printCallback, format, args) catch unreachable;
+    fmt.format({}, TtyError, printCallback, format, args) catch |_| {
+        log.logError("TTY: Error printing\n");
```
Comment (SamTebbs33) [inline src/kernel/tty.zig]: Would be nice to print the error name with `@errorName`. Same with a few more places in this file.

File: test/mock/kernel/vga_mock.zig (hunk 1 above)
Comment (DrDeano) [reply test/mock/kernel/vga_mock.zig]: I documented in one of the wiki files that if the mocked function has the same behavior as the source function, then prefix with `orig_`.

File: src/kernel/tty.zig [hunk 3]
```diff
@@ -361,55 +387,58 @@ fn writeString(str: []const u8) PrintError!void {
...
     column = 0;
     row = 0;
 
-    writeString("                  _____    _        _    _   _______    ____  \n") catch unreachable;
-    writeString("                 |  __ \\  | |      | |  | | |__   __|  / __ \\ \n") catch unreachable;
-    writeString("                 | |__) | | |      | |  | |    | |    | |  | |\n") catch unreachable;
-    writeString("                 |  ___/  | |      | |  | |    | |    | |  | |\n") catch unreachable;
-    writeString("                 | |      | |____  | |__| |    | |    | |__| |\n") catch unreachable;
-    writeString("                 |_|      |______|  \\____/     |_|     \\____/ \n") catch unreachable;
+    writeString(logo1 ++ logo2 ++ logo3 ++ logo4 ++ logo5 ++ logo6) catch |_| {
```
[the remark this answers is not in the document]
Comment (DrDeano) [reply src/kernel/tty.zig]: Can do :)

File: src/kernel/tty.zig (hunk 2 above)
Comment (DrDeano) [reply src/kernel/tty.zig]: I was thinking of this, will add printing the error

File: test/mock/kernel/vga_mock.zig (hunk 1 above)
Comment (SamTebbs33) [reply test/mock/kernel/vga_mock.zig]: I see

**Pull request merged**